0004-Update-releases-in-appdata.patch

```diff
@@ -1,30 +0,0 @@
-From 46c0d9750bf1599a9823d87838fc0935291ebaca Mon Sep 17 00:00:00 2001
-From: Ryan Gonzalez <rymg19@gmail.com>
-Date: Fri, 9 Jul 2021 12:07:38 -0500
-Subject: [PATCH 4/5] Update releases in appdata
```

Comment:
Why combine this and the other patch? Having them separate is useful to update only one without introducing weird conflicts.

Reply:
Because both were changes in the appdata file, 0005- doesn't need to updated, only the release section needs to updated. I can separate them if you want.

Reply:
Seperated